project/app/utils/moderation/text_moderation.py

```diff
@@ -0,0 +1,71 @@
+import io
+from os import path
+
+
+class TextModerationBase:
+    """Base Class for text moderation classes"""
+    def __init__(self, wordlist_file: str):
+        """Arguments:
+        `wordlist_file` str - path to file containing words to be
+        used in moderation"""
```

Comment:
Can you change to "words to be moderated" rather than "words to be used in moderation" to avoid chance of confusion?